### PR TITLE
Restart winrm

### PR DESCRIPTION
**Pull request by Plork**
add the option to disable the restart of winrm with admin center install.

### CODE REVIEW

**Comment by flcdrg** [comment]
Thanks for the PR!

Given I presume you'd only want to use this to disable the restart, I'm thinking it might be nicer to make this an 'opt-in' parameter..

eg call it '\DisableRestartWinRM` and only add it to the args list if it is passed in:

```
if ($pp['RESTARTWINRM']) {
    $restart_winrm = "RESTART_WINRM=0"
}
```

**Comment by Plork** [comment]
fine with me, thats the only discussion I was looking for ;) was exactly my internal debat.

naming parameters is alway the hardest thing :D

Will change the script

**Comment by Plork** [comment]
I rewrote arguments to be more clear for optional flags.
also less struggles with quotes like this.

apologies for my editor removing trailing spaces :S

**Comment by flcdrg** [comment]
I tested this out, but it failed. Missing a space I think

```text
ERROR: Running ["C:\Windows\System32\msiexec.exe" /i "C:\Users\david\AppData\Local\Temp\chocolatey\windows-admin-center\1.3.44065.0\WindowsAdminCenter2110.msi" /qn norestart SME_PORT=6516 SSL_CERTIFICATE_OPTION=generate RESTART_WINRM=0/l*v "C:\Users\david\AppData\Local\Temp\chocolatey\windows-admin-center.1.3.44065.0.MsiInstall.log" ] was not successful. Exit code was '1639'. Exit code indicates the following: The command line arguments passed to the MSI are incorrect. If you passed in additional arguments, please adjust. Otherwise followup with the package maintainer(s) to get this fixed. This is most likely an issue with the 'windows-admin-center' package and not with Chocolatey itself. Please follow up with the package maintainer(s) directly..
The install of windows-admin-center was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\windows-admin-center\tools\chocolateyinstall.ps1'.
 See log for details.
```

File: windows-admin-center/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
+
+if ($pp['THUMBPRINT']) {
+  $arguments += ' SME_THUMBPRINT={0}' -f $pp['THUMBPRINT']
+}
+
+if ($pp['DISABLERESTARTWINRM']) {
+  $arguments += ' RESTART_WINRM=0'
+}
+
+$arguments += '/l*v "{0}\{1}.{2}.MsiInstall.log"' -f $env:Temp, $packageName, $env:chocolateyPackageVersion
```
Comment (flcdrg) [inline windows-admin-center/tools/chocolateyinstall.ps1:35]:
```suggestion
$arguments += ' /l*v "{0}\{1}.{2}.MsiInstall.log"' -f $env:Temp, $packageName, $env:chocolateyPackageVersion
```

File: windows-admin-center/windows-admin-center.nuspec [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
+Windows Server, version 1709 	Gateway mode
+Windows Server 2016 	Gateway mode
+
+### Package Specific
+#### Package Parameters
+The following package parameters can be set:
+
+ * `/Port:` - determines what port to listen on. Defaults to 6516. (You may want to use 443 on Windows Server)
+ * `/Thumbprint:` - Existing certificate thumbprint. If not specified, a self-signed certificate is generated (expires after 60 days)
+ * `/DisableRestartWinrm:` -  If you are installing with an automation solution that would be broken by the WinRM service restarting, you can disable this by passing this parameter
```
Comment (flcdrg) [inline windows-admin-center/windows-admin-center.nuspec:38]:
```suggestion
 * `/DisableRestartWinrm` -  If you are installing with an automation solution that would be broken by the WinRM service restarting, you can disable this by passing this parameter
```

**Review by flcdrg — CHANGES_REQUESTED** [review]

**Comment by Plork** [comment]
thnx for the test!! I missed those

**Review by flcdrg — APPROVED** [review]

**Comment by flcdrg** [comment]
I've pushed a new version with this change. Hopefully it will appear soon

**Pull request merged**